app/src/main/java/com/glia/exampleapp/MainFragment.kt

```diff
@@ -458,14 +458,14 @@ class MainFragment : Fragment() {
         val visitorContext = getContextAssetIdFromPrefs(sharedPreferences)
         val cv = GliaWidgets.getCallVisualizer()
         if (!visitorContext?.trim { it <= ' ' }.isNullOrBlank()) {
-            cv.addVisitorContext(visitorContext)
+            cv.addVisitorContext(visitorContext!!)
```

Comment:
I would prefer to avoid force unwrapping. I would rather use the let{ } construction.

Reply:
fixed [b2be851](https://github.com/salemove/android-sdk-widgets/pull/885/commits/b2be85199ae2ef8ac3475fbcd2177a406be6b3fc)